packages/tools/client-debugger/client-debugger/package.json

```diff
@@ -92,11 +92,18 @@
 		"eslint-plugin-chai-expect": "~3.0.0",
 		"mocha": "^10.0.0",
 		"nyc": "^15.0.0",
+		"jest": "^26.6.3",
```

Comment:
This package shouldn't need jest. We're using `mocha` for testing in this package, since we don't do anything with UI / the browser. We should probably revert these changes.

Reply:
I used `jest`'s not null to verify when new container generated, debugger size will change and audience is not null. I didn't find `mocha` has not null/empty verify.

Reply:
The `chai` library has the `expect` library we use when we're using mocha. It should have functionality like `expect(foo).to.not.be.null`. It's syntactically different from `jest`'s expect, but it should do the same thing.

Reply:
I see! Modifying it. Thx!